sql/src/main/java/io/crate/analyze/FrameBoundDefinition.java

```diff
@@ -65,6 +66,17 @@ public Symbol value() {
         return value;
     }
 
+    public FrameBoundDefinition map(Function<? super Symbol, ? extends Symbol> mapSymbolsFunction) {
```

Comment:
Minor: Might want to go with `mapper` for the function, for consistency with other `map` functions. But feel free to leave as is.